index.bs

```diff
@@ -201,13 +201,13 @@ The <dfn attribute for="XRSession">environmentBlendMode</dfn> attribute MUST rep
 XR Compositor Behaviors {#xr-compositor-behaviors}
 ---------------------
 
-When presenting content to the [=XR device=], the [=XR Compositor=] MUST apply the appropriate <dfn>blend technique</dfn> to combine virtual pixels with the [=real-world environment=]. The appropriate technique is determined based on the [=XR device=]'s [=display technology=] and the [=XRSession/mode=].
+When presenting content to the [=XR device=], the [=XR Compositor=] MUST apply the appropriate <dfn>blend technique</dfn> to combine virtual pixels with the [=real-world environment=]. The appropriate technique is determined based on the [=XR device=]'s [=display technology=] and the [=XRSession/mode=]. The [=XR Compositor=] MAY assume that all alpha values are premultiplied; this specification does not define the blending behavior for layers that contain pixels without premultiplied alpha.
 
 - When performing <dfn>opaque environment blending</dfn>, the rendered buffers obtained by the [=XR Compositor=] are composited using [=source-over=] blending on top of buffers containing exclusively 100% opaque black pixels. The composited output is then presented on the [=XR device=]. This technique MUST be applied on [=opaque=] and [=pass-through=] displays when the [=XRSession/mode=] is set to either {{XRSessionMode/"immersive-vr"}} or {{XRSessionMode/"inline"}}. This technique MUST NOT be applied when the [=XRSession/mode=] is set to {{XRSessionMode/"immersive-ar"}}, regardless of the [=XR Device=]'s [=display technology=].
 
 - When performing <dfn>alpha-blend environment blending</dfn>, the rendered buffers obtained by the [=XR Compositor=] are composited using [=source-over=] blending on top of buffers containing pixel representations of the [=real-world environment=]. These pixel representations must be aligned on each {{XRFrame}} to the {{XRView/transform}} of each view in {{XRViewerPose/views}}. The composited output is then presented on the [=XR device=]. This technique MUST be applied on [=pass-through=] displays when the [=XRSession/mode=] is set {{XRSessionMode/"immersive-ar"}}. This technique MUST NOT be applied when the [=XRSession/mode=] is set to {{XRSessionMode/"immersive-vr"}} or {{XRSessionMode/"inline"}} regardless of the [=XR Device=]'s [=display technology=].
 
-- When performing <dfn>additive environment blending</dfn>, the rendered buffers obtained by the [=XR Compositor=] are composited using [=lighter=] blending before being presented on the [=XR device=]. This technique MUST be applied on [=additive light=] displays, regardless of the [=XRSession/mode=].
+- When performing <dfn>additive environment blending</dfn>, the rendered buffers obtained by the [=XR Compositor=] are directly presented on the [=XR device=]. This is equivalent to them being composited with the real world using [=lighter=] blending. This technique MUST be applied on [=additive light=] displays, regardless of the [=XRSession/mode=].
```

Comment:
This line is currently wrong in the spec. Layers are composited with source-over in all cases.
I filed on issue on this. For now, maybe it's better not to change this.

Reply:
Right, this is an attempt to fix that issue. We don't currently composite layers with each other in any way.

What changes would you like to see to address #19, then?

Reply:
My desire here is to avoid specifying behavior for the case of multiple layers when that case isn't yet supported. We can mention source-over blending in the additive mode once we actually have multiple layers being composited with each other, in the meantime your comments in #23 indicate that we should mention that the layer is composited with the real world with `lighter`.